docs/_includes/date-picker.html

```diff
@@ -0,0 +1,144 @@
+<div id="date-picker">
+    {% include section-header.html name="date-picker" version=page.versions.date-picker %}
+
+    <p>The date picker component is used to display a calendar from which dates can be selected, either as single items or as a range.</p>
+    <p>Arrow keys should navigate between dates, but this is an implementation detail that is not yet included in skin.</p>
+
+    <h3>Single Pane</h3>
+
+    <div class="date-picker" aria-label="date picker">
+        <div class="date-picker__header">
+            <div class="date-picker__header--inner">
+                <button
+                    class="icon-btn icon-btn--small icon-btn--transparent"
+                    aria-label="Previous Month - July 2019"
+                    variant="icon"
+                >
```

Comment:
Typo. Moving closing bracket to end of previous line.

Reply:
Oh, I see how you are formatting it now. Generally though we are not following that convention.